dyno-core/src/main/java/com/netflix/dyno/connectionpool/impl/lb/AbstractTokenMapSupplier.java

```diff
@@ -218,12 +227,13 @@ private boolean isLocalDatacenterHost(Host host) {
                 String zone = (String) jItem.get("zone");
                 String datacenter = (String) jItem.get("dc");
                 String portStr = (String) jItem.get("port");
+                String hashtag = (String) jItem.get("hashtag");
```

Comment:
Just curious, what happens if there is no hashtag field from token map supplier? Have you tested it.?

Reply:
I am not following the question. If no hashtag is provided, then it will work as before. Did I test it? Yes, I made sure all the vanilla test cases work. The idea is for everything to be backward compatible.